rank/docs/arabic-examples.md

```diff
@@ -1,8 +0,0 @@
-# Arabic examples
-
-These examples should be added to rank when the TEI work has been merged, to make sure we're analysing arabic text correctly.
-
-| search term | matched term in document | work                                                                                                          |
-| ----------- | ------------------------ | ------------------------------------------------------------------------------------------------------------- |
-| لكشف        | الكشف                    | https://github.com/wellcomecollection/wellcome-collection-tei/blob/master/Arabic/WMS_Arabic_256.xml           |
```

Comment:
Is it worth putting these links somewhere in the test config? They seem like useful context.

Reply:
Maybe... 

We already include the work ID and a short description of the match we're trying to make, and we haven't needed quick access source data in the course of the work so far. We usually think of the target works as unchangeable from rank's POV, but knowledge of source works is worth keeping in mind with the ongoing questions about matching/merging. 

I'm going to leave it out of this PR but might come back to add source works more comprehensively across our test cases another time.

Reply:
pronge @jamesgorrie for any thoughts on ☝️

Reply:
What are the links for? As far as I can see it's an example, but now the work is in the API, we'll have the link from the ratings which is the work ID?